src/test/java/pl/simpleascoding/tutoringplatform/user/UserServiceImplTest.java

```diff
@@ -107,37 +88,36 @@ void whenGetUserByUsername_thenUserNotFoundExceptionShouldBeThrown() {
         assertThrows(UserNotFoundException.class, () -> userServiceImpl.getUserByUsername(USERNAME));
     }
 
-    @DisplayName("Should return null and User registration confirmed.")
+    @DisplayName("confirmUserRegistration")
     @Test
     void whenConfirmUserRegistration_thenUserRegistrationConfirmedShouldBeReturned() {
         //given
         Token tokenEntity = createTokenEntity();
 
         when(tokenRepository.findTokenByValue(tokenEntity.getValue())).thenReturn(Optional.of(tokenEntity));
         //when
-        RscpDTO<?> resultRscpDTO = userService.confirmUserRegistration(tokenEntity.getValue());
+        RscpDTO<?> resultRscpDTO = userServiceImpl.confirmUserRegistration(tokenEntity.getValue());
 
         RscpDTO<Object> expectedRscpDTO = new RscpDTO<>(RscpStatus.OK, "User registration confirmed.", null);
 
         assertThat(resultRscpDTO, is(equalTo(expectedRscpDTO)));
     }
```

Comment:
I would add Mockito.verify() as static in short form just in case verify(), a method verifying whether tokenRepository was called once